perseo-utils/pom.xml

```diff
@@ -47,7 +47,7 @@
         <dependency>
             <groupId>log4j</groupId>
             <artifactId>log4j</artifactId>
-            <version>1.2.17</version>
+            <version>1.2.27</version>
```

Comment:
Maybe shoul be documented in the CNR (as for instances: "Upgrade Esper library from 6.1.0 to 7.1.0")